queries/ruby/highlights.scm

```diff
@@ -16,6 +16,14 @@
   "then"
 ] @keyword
 
+(begin_block) @keyword
+
+(end_block) @keyword
```

Comment:
```suggestion
[ 
  "BEGIN"
  "END"
] @keyword
```
Otherwise the entire block will be given the keyword highlight, rather than just the keywords themselves